src/lib/data/wish-setup.json

```diff
@@ -1,6 +1,6 @@
 {
 	"version": "3.5",
-	"wishPhase": 1,
+	"wishPhase": 2,
 	"storageVersion": "3.5.0",
```

Comment:
`storageVersion` needs to be changed to automatically apply updates to site.

Reply:
Yeah, it's okay.
Actually it doesn't matter if you want to change it to any number, the site will match with the stored data, if this property does not change, the site will not update automatically.